src/table/context/TableContext.tsx

```diff
@@ -51,6 +52,10 @@ export const TableContextProvider = ({
     tableWidth,
     styling
   );
+  const isBorderRight = useMemo(
```

Comment:
Not sure the name makes sense, showRightBorder or hasRightBorder is more descriptive